core/commands/storage.go

```diff
@@ -112,61 +113,123 @@ Receive proofs as collateral evidence after selected nodes agree to store the fi
 		cmds.StringArg("file-hash", true, true, "Add hash of file to upload.").EnableStdin(),
 	},
 	Options: []cmds.Option{
+		cmds.BoolOption(leafHashOptionName, "l", "Flag to specify given hash(es) is leaf hash(es).").WithDefault(false),
```

Comment:
Can you also update the Helptext above (example usage)? I think it's out of date ..